docs/references/chrome-extension/sync-host.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Sync your Chrome Extension with your web app
```

Comment:
Same here as in https://github.com/clerk/clerk-docs/pull/1004/commits/ff92b58598243b8e08fb9704303a766619840ded#r1809419432

With this page I'm just thinking of making the title clearer for people using search or skimming the side bar.

Reply:
can do!

Reply:
updated! [b089102](https://github.com/clerk/clerk-docs/pull/1004/commits/b089102b7f14df60439d79fd7c6313e1a735050a)